src/common/config_opts.h

```diff
@@ -1211,7 +1211,7 @@ OPTION(rbd_default_format, OPT_INT, 2)
 OPTION(rbd_default_order, OPT_INT, 22)
 OPTION(rbd_default_stripe_count, OPT_U64, 0) // changing requires stripingv2 feature
 OPTION(rbd_default_stripe_unit, OPT_U64, 0) // changing to non-object size requires stripingv2 feature
-OPTION(rbd_default_features, OPT_INT, 61)   // only applies to format 2 images
+OPTION(rbd_default_features, OPT_STR, "layering,exclusive lock,object map,fast-diff,deep-flatten")   // only applies to format 2 images
```

Comment:
It would be nice to keep the names consistent between the config and the rbd CLI (i.e. exclusive lock -> exclusive-lock).  I would recommend creating constants in src/include/rbd/features.h for each of the features that both the rbd CLI and librbd can utilize.